carbon-design-system/carbon-addons-iot-react
# fix(table): hide multisort options in rowEdit to match current sort

Closes #2863 

**Summary**

- When in rowEdit mode the sort options are hidden, but those for multi-sort were still visible. This removes them when in rowEdit mode to match behavior of singular sort.

**Change List (commits, features, bugs, etc)**

- add checks for `isDisabled` prop and hide multi-sort options/classes when necessary
- added MultiSort knob to rowEdit table story to verify

**Acceptance Test (how to verify the PR)**

- open the Table with full rowEdit example
- turn on MultiSort
- click the edit button in the toolbar to turn on rowEdit mode
- confirm the sort icons/numbers/multisort overflow menu is hidden

**Regression Test (how to make sure this PR doesn't break old functionality)**

- check other MultiSort stories for inconsistencies



**Things to look for during review**
